sample/src/main/java/com/example/bottomnavigator/MainActivity.kt

```diff
@@ -75,7 +75,13 @@ class SampleFragment : Fragment() {
 
 
         rootView.findViewById<Button>(R.id.btn).setOnClickListener {
-            bottomNavigator.addFragment(SampleFragment())
+            bottomNavigator.addFragment(
```

Comment:
Can you add a separate button with "Add Fragment with animation". Since this sample serves as documentation I would like to keep an example with the simple call to addFragment.

Reply:
Good idea 👍